GeneralsMD/Code/GameEngine/Source/GameClient/MapUtil.cpp

```diff
@@ -1156,7 +1178,12 @@ Image *getMapPreviewImage( AsciiString mapName )
 	tgaName.removeLastChar(); // m
 	tgaName.removeLastChar(); // .
 	name = tgaName;//.reverseFind('\\') + 1;
-	filename = tgaName.reverseFind('\\') + 1;
+	if(tgaName.reverseFind('\\')) {
+		filename = tgaName.reverseFind('\\') + 1;
+	}
+	else if(tgaName.reverseFind('/')) {
+		filename = tgaName.reverseFind('/') + 1;
+	}
```

Comment:
It looks like there's cases where this path can have either windows or linux, so to be safe, checking for both.